packages/back-end/src/jobs/updateExperimentResults.ts

```diff
@@ -93,8 +98,18 @@ export default async function (agenda: Agenda) {
     organization: string,
     experimentId: string
   ) {
+    const org = await findOrganizationById(organization);
+
+    if (!org) return;
+
+    const context: ApiReqContext = {
+      org,
+      environments: getEnvironmentIdsFromOrg(org),
+      readAccessFilter: FULL_ACCESS_PERMISSIONS,
+    };
+
     const job = agenda.create(UPDATE_SINGLE_EXP, {
-      organization,
+      context,
```

Comment:
I think this change requires an accompanying change to the `job.unique` call below. (Might want to double-check with Jeremy on this.)

Something like:
```
job.unique({
  experimentId,
  context: {
    organization: { id: org.id }
  },
});
```

Reply:
Looking at AgendaJS' documentation, it looks like this ensure that only one instance of this job exists with the specified properties, so I think this should be sufficient. But I will double check with Jeremy just to be safe.

Reply:
Jeremy had feedback to update these jobs so we continue just passing in the organization when queing, and when executing the job, we build the `context` object from the `orgId`.